# bioinformateachers
Code archive linked to the bioinformatics-oriented educational website [The Bioinformateachers](https://bioinformateachers.github.io/)

- [Naive Bayes classifier â€“ a naive introduction (with R)](https://bioinformateachers.github.io/statistics/classification/bayes/r/2022/06/22/naive-bayes.html)
- [Amplicon sequencing: a quick look at the construct](https://bioinformateachers.github.io/bioinformatics/microbiome/2022/06/11/quick-ampliseq.html)
- [R tip: remove the last character from an array of strings](https://bioinformateachers.wordpress.com/2022/05/25/r-tip-remove-the-last-character-from-an-array-of-strings/)
- [Super-quick recap of linear mixed models](https://bioinformateachers.github.io/statistics/quantitative-genetics/linear-algebra/2022/01/05/linear-mixed-models.html)

# Deep Learning for Life Sciences [book]
###  (with applications in Python)

In the folder [dlb](dlb/) we store a collection of Python notebooks linked to the book: 
these notebooks are meant to provide code and exercise to illustrate some of the concepts 
covered in the book:

1. [K-means clustering](dlb/archipelago/lab_full.ipynb)
2. Confusion matrix [N.]
3. ROC, AUC, MCC [F.]
4. Additional model performance metrics [N.]
5. [Backpropagation](dlb/backpropagation/backpropagation.ipynb)
6. [Function approximation](dlb/function_approximation/function_approximation.ipynb)
7. [Logistic regression with neural networks](dlb/regression/logistic_regression.ipynb)
8. [Softmax regression with neural networks](dlb/regression/softmax_regression.ipynb)
9. Linear regression with neural networks [F.]
10. From basic regression to deeper NN models [F.]
11. Basic RNN [F.]
12. Specialized RNN [F.]
13. Data augmentation [N.]
14. Embeddings [F.]
15. Transfer learning [N.]
16. Feature selection [N.]
17. Opening the black box [N.]
18. Transformers? [F.]

