# Notes on alfalfa (medicago sativa)

* [2020, Chen et al] [Allele-aware chromosome-level genome assembly and efficient 
  transgene-free genome editing for the autotetraploid cultivated alfalfa](https://www.nature.com/articles/s41467-020-16338-x)
  * chromosome-aware full sequencing of chinese cultivar XinJiangDaYe
  * sequencing:
	* 70 gigabases (Gb) of PacBio CCS long reads, Sequel platform
	* 126 Gb of Illumina short reads,HiSeq2000 platform
	* 99 Gb ONT (Oxford Nanopore Technology) long reads with average reads length of 16 Kb
  * very good details on sequencing, assembling, quality testing
  * genome and transcriptome sequencing raw data are on NCBI, project [PRJNA540215](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA540215)
  * genome assembly files [available on figshare](https://figshare.com/projects/whole_genome_sequencing_and_assembly_of_Medicago_sativa/66380)


## To be investigated (stuff that I haven't read, yet)

* [2022] [Genome Assembly of Alfalfa Cultivar Zhongmu-4 and Identification of SNPs Associated with Agronomic Traits](https://www.sciencedirect.com/science/article/pii/S1672022922000031)
* [2014] [Development of an Alfalfa SNP Array and Its Use to Evaluate Patterns of Population Structure and Linkage Disequilibrium](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0084329)
  * SNP array for alfalfa
* [2022] [Application of machine learning to explore the genomic prediction accuracy of fall dormancy in autotetraploid alfalfa](https://academic.oup.com/hr/article/10/1/uhac225/6751727?login=false)

