# Notes on alfalfa (medicago sativa)

* [2020, Chen et al.] [Allele-aware chromosome-level genome assembly and efficient 
  transgene-free genome editing for the autotetraploid cultivated alfalfa](https://www.nature.com/articles/s41467-020-16338-x)
	* chromosome-aware full sequencing of chinese cultivar XinJiangDaYe
	* sequencing:
		* 70 gigabases (Gb) of PacBio CCS long reads, Sequel platform
		* 126 Gb of Illumina short reads,HiSeq2000 platform
		* 99 Gb ONT (Oxford Nanopore Technology) long reads with average reads length of 16 Kb
	* very good details on sequencing, assembling, quality testing
	* genome and transcriptome sequencing raw data are on NCBI, project [PRJNA540215](https://www.ncbi.nlm.nih.gov/bioproject/PRJNA540215)
	* genome assembly files [available on figshare](https://figshare.com/projects/whole_genome_sequencing_and_assembly_of_Medicago_sativa/66380)
* [2022, Long et al.] [Genome Assembly of Alfalfa Cultivar Zhongmu-4 and Identification of SNPs Associated with Agronomic Traits](https://www.sciencedirect.com/science/article/pii/S1672022922000031)
	* chromosome aware full sequencing of chinese cultivar Zhongmu-4
	* comprehensive database of genomic variations based on resequencing of 220 germplasms
	* using the 168 Gb Illumina 150 bp paired-end genome sequencing reads, the Zhongmu-4 
	genome was surveyed and the tetraploid genome size was estimated as 2962 Mb by KmerGenie according to the K-mer frequency (K = 19). 
	* the estimated Zhongmu-4 genome size was set at 3.1 Gb for further analyses.
	* sequencing:
		* 262 Gb continuous long reads (CLRs) generated by the PacBio Sequel sequencing system
		* 168 Gb Illumina paired-end sequences (short reads)
	* Repetitive elements drive genome expansion of alfalfa
	* The genome assembly is available on [figshare](https://figshare.com/s/fb4ba8e0b871007a9e6c) and on the [chinese genome warehouse](https://ngdc.cncb.ac.cn/gwh/)


## To be investigated (stuff that I haven't read, yet)

* [2014] [Development of an Alfalfa SNP Array and Its Use to Evaluate Patterns of Population Structure and Linkage Disequilibrium](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0084329)
  * SNP array for alfalfa
* [2022] [Application of machine learning to explore the genomic prediction accuracy of fall dormancy in autotetraploid alfalfa](https://academic.oup.com/hr/article/10/1/uhac225/6751727?login=false)
* [2020, Shen et al.] [The chromosome-level genome sequence of the autotetraploid alfalfa and resequencing of core germplasms provide genomic resources for alfalfa research](https://www.sciencedirect.com/science/article/pii/S1674205220302161)
	* cultivar: Zhongmu-1
	* assembled at the haploid chromosome level
